### PR TITLE
Adding proper error message when deleting a role

### PR DESCRIPTION
**Pull request by AnjanaSamindraPerera**
### Related Issues
- https://github.com/wso2/product-is/issues/17897

### CODE REVIEW

File: .changeset/grumpy-poems-deliver.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/i18n": patch
```
Comment (dasuni-30) [inline .changeset/grumpy-poems-deliver.md:2]: Let's add the `console` and `myaccount` as well.

Comment (jeradrutnam) [reply .changeset/grumpy-poems-deliver.md:2]: `console` will be enough. As it's console specific change.

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 2]
```diff
@@ -10602,7 +10602,7 @@ export const console: ConsoleNS = {
                     deleteRole: {
                         error: {
                             description: "{{description}}",
-                            message: "Error deleting the selected role."
+                            message: "Unable to delete the selected role."
```
Comment (jeradrutnam) [inline modules/i18n/src/translations/en-US/portals/console.ts:10605]: Update `fr-FR` and `si-LK` as well

File: .changeset/grumpy-poems-deliver.md (hunk 1 above)
Comment (dasuni-30) [reply .changeset/grumpy-poems-deliver.md:2]: We need to bump both `console` and `myaccount` to maintain the consistency of versioning.

Comment (AnjanaSamindraPerera) [reply .changeset/grumpy-poems-deliver.md]: resolved in https://github.com/wso2/identity-apps/pull/4727/commits/b533228182a5092b13584912e7032c1147eacc69

**Review by dasuni-30 — APPROVED** [review]

**Pull request merged**